internal/commands/token/rm.go

```diff
@@ -67,11 +67,12 @@ func runRemove(streams command.Streams, hubClient *hub.Client, opts removeOption
 	}
 
 	if !opts.force {
+
 		fmt.Fprintf(streams.Out(), ansi.Warn("WARNING: This action is irreversible.")+`
-By confirming, you will permanently delete the access token.
-Deleting a token will invalidate your credentials on all Docker clients currently authenticated with this token.
+By confirming, you will permanently revoke and delete the access token.
```

Comment:
About rephrasing the warning message, the original intent was to match Docker Hub UX:
<img width="741" alt="Screenshot 2022-04-13 at 09 55 07" src="https://user-images.githubusercontent.com/31478878/163128180-018f71fe-607f-4137-b9af-bfd66289711a.png">